programmers/unfinished-runner.js

```diff
@@ -0,0 +1,22 @@
+const participant = ["leo","kiki","eden","leo"];
+const completion = ["eden", "kiki"];
+function solution(participant, completion) {
+    let  counts = {};
+    participant.forEach(function(i){
+        if (i in counts){
+            counts[i] += 1;
+        }
+        else{
+            counts[i] = 1;
+        }
+    })
+    completion.forEach(function(j){
+        counts[j] -=1;
+        if (counts[j]===0){
+            delete counts[j];
+        }
+    })
+    return Object.keys(counts);
```

Comment:
리턴하는 값이 [] 배열안에 들어가서 리턴을 하고 있으므로 join을 이용해서 문자열로 리턴해줍니다. 문제에서 요구하는 리턴값은 문자열입니다.
너무 잘 푸셔서 딱히 코멘트 드릴게없습니다.
```suggestion
    return Object.keys(counts).join('');
```